docs/ref/urlresolvers.txt

```diff
@@ -70,9 +70,8 @@ use for reversing. By default, the root URLconf for the current thread is used.
         >>> reverse('cities', args=['Orléans'])
         '.../Orl%C3%A9ans/'
 
-    Applying further encoding (such as :meth:`~django.utils.http.urlquote` or
-    ``urllib.quote``) to the output of ``reverse()`` may produce undesirable
-    results.
+    Applying further encoding (such as :py:func:`urllib.parse.quote`) to the
```

Comment:
I think the `:py` prefix isn't needed (at least, there are many more links without it than with it in Django's docs).